### PR TITLE
Add mTLS examples.md

### PR DESCRIPTION
**Pull request by danielforsap**
# Pull Request Template

## Prerequisites

* If there is no issue related to your pull request - open one and assign yourself to it. If you're proposing a solution to an already opened issue - simply assign yourself to it.

## Motivation

*Describe the problem you're trying to solve or the feature you're proposing in addition to a link to the related issue.*

## Approach

_How does this change address the problem?_

## Pull Request status

* [x] Initial implementation
* [ ] Refactoring
* [ ] Unit tests
* [ ] Integration tests

Feel free to construct the checklist with whatever items seem most reasonable for your change. You could disassemble the Implementation part to even smaller separate checklist items if you're working on something big for example. But do make the effort to provide a checklist of some sort so that the core team as well as the community can have an idea about the progress of your Pull Request.

## Third-party code

If you use third party code with your contribution such as, components, libraries, or snippets make sure to mention that.

## Work in Progress label

For as long as development of your Pull Request is still ongoing you MUST label it with a **wip** label as well as prefix the name of the PR with *[WIP]*.

For example: *[WIP] Service brokers API*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,7 +274,22 @@ stringData:
...
+  name: sap-btp-service-operator
+  namespace: sap-btp-operator
+type: kubernetes.io/tls
+stringdata:
+  clientid: <clientid>
+  certificate: <certificate>
+  key: <key>
+  sm_url: <sm_url>
+  tokenurl: <auth_url>
+```
```
Comment (I065450) [inline README.md:292]:
```yaml
apiVersion: v1
kind: Secret
metadata:
  name: sap-btp-service-operator
  namespace: sap-btp-operator
type: Opaque
stringdata:
  clientid: <clientid>
  tls.crt: <certificate>
  tls.key: <key>
  sm_url: <sm_url>
  tokenurl: <auth_url>
  tokenurlsuffix: "/oauth/token"
```

Comment (danielforsap) [reply README.md:292]: @I065450 I wanted to be sure the example was ok before I added it here too.

Comment (danielforsap) [reply README.md:281]: @I065450 changed.

**Review by danielforsap — COMMENTED** [review]
Changed

File: README.md [hunk 2]
```diff
@@ -268,8 +268,8 @@ mTLS Access Credentials
 apiVersion: v1
 kind: Secret
 metadata:
-  name: sap-btp-service-operator
-  namespace: sap-btp-operator
+  name: mybtpsecret
```
Comment (danielforsap) [inline README.md:271]: @I065450  not sure about this name for the secret

**Comment by danielforsap** [comment]
@avilupu  can you please review this PR?
Naomi already reviewed, added some changes, I changed the rest.

Thanks

**Review by I065450 — APPROVED** [review]

**Pull request merged**